manifests/init.pp

```diff
@@ -203,6 +203,8 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
+# $yum_max_speed::              Maximum bandwidth used per download in bytes/sec
```

Comment:
right, i guess i'm thinking that most people use this to limit their bandwidth and might not know what 'per task' means, granted they likely wouldn't know what 'per worker' means either, but the maximum it would take up is the speed times the number of workers (which would be the number of tasks executing at once)